oem/ibm/configurations/dbus-config.json

```diff
@@ -0,0 +1,6 @@
+{
+    "entityTypes": [
+        32903,
+        64
```

Comment:
you can remove this.we dont need to restore this.

Reply:
> you can remove this.we dont need to restore this.

Done